crates/sui-indexer-alt-consistent-store/src/db/mod.rs

```diff
@@ -105,6 +108,34 @@ struct IterBounds<'d>(
     Option<rocksdb::DBRawIterator<'d>>,
 );
 
+/// Metrics related to memory usage and backpressure for a column family.
+pub struct ColumnFamilyMetrics {
```

Comment:
It would be good to have more explanation here on what these metrics are tracking.